### PR TITLE
Add Serbo-croate language name

### LINKED ISSUE #889: Serbo-croate language does not appear properly in kiwix-serve language selection box
This is what we have currently:
image

In place of `hbs` we should read something like srpskohrvatski (https://en.wikipedia.org/wiki/Serbo-Croatian)

### PR DESCRIPTION
**Pull request by juuz0**
Fixes #889

### CODE REVIEW

**Review by mgautierfr — CHANGES_REQUESTED** [review]
A very small change to help in the next PR. #pedantic

File: src/opds_dumper.cpp [hunk 1]
```diff
@@ -165,7 +165,8 @@ std::map<std::string, std::string> iso639_3 = {
   {"rmr", "Caló"},
   {"rmy", "romani shib"},
   {"roa", "romance languages"},
-  {"twi", "twi"}
+  {"twi", "twi"},
+  {"hbs", "srpskohrvatski"}
```
[the remark this answers is not in the document]
Comment (juuz0) [reply src/opds_dumper.cpp]: Thanks, done :)

File: src/opds_dumper.cpp [hunk 2]
```diff
@@ -165,7 +165,8 @@ std::map<std::string, std::string> iso639_3 = {
   {"rmr", "Caló"},
   {"rmy", "romani shib"},
   {"roa", "romance languages"},
-  {"twi", "twi"}
+  {"twi", "twi"},
+  {"hbs", "srpskohrvatski"},
```
Comment (veloman-yunkan) [inline src/opds_dumper.cpp:169]: This list appears to be sorted for convenience. I don't mind if it's not sorted but a general good habit is to respect conventions.

Comment (juuz0) [reply src/opds_dumper.cpp]: Eagle eye :)

Kept the comma from mgautierfr's comment.

**Review by veloman-yunkan — APPROVED** [review]

File: src/opds_dumper.cpp [hunk 3]
```diff
@@ -165,7 +166,7 @@ std::map<std::string, std::string> iso639_3 = {
   {"rmr", "Caló"},
   {"rmy", "romani shib"},
   {"roa", "romance languages"},
-  {"twi", "twi"}
+  {"twi", "twi"},
```
Comment (kelson42) [inline src/opds_dumper.cpp:169]: We should not have  a comma here

**Review by kelson42 — CHANGES_REQUESTED** [review]

File: src/opds_dumper.cpp (hunk 3 above)
Comment (kelson42) [reply src/opds_dumper.cpp:169]: My bad, was a request from @mgautierfr

**Pull request merged**